src/commtrack.js

```diff
@@ -136,10 +137,14 @@ define([
             isHashtaggable: false,
             spec: {
                 date: {
-                    visibility: 'hidden',
+                    lstring: 'Date',
+                    visibility: 'visible',
                     presence: 'optional',
-                    serialize: function () {},
-                    deserialize: function () {},
+                    widget: widgets.xPath,
+                    xpathType: 'generic',
+                    serialize: mugs.serializeXPath,
+                    deserialize: mugs.deserializeXPath,
+                    help: 'The date and time of the action, e.g., now() or today()',
```

Comment:
I kept today() in there since you mentioned a project that wanted to use dates as their date reported, although I think they would have been using the input from another question. When you create the question it fills in now() automatically.